js/main.js

```diff
@@ -26,7 +26,8 @@ jQuery(function($) {
 				var $this = $(this);
 
 				// add a tab head link for each tab
-				tabsHead.append('<a href="#">' + $this.find('.gsection_title').html() + '</a>');
+				// add the id as well so gf conditional sections still works
+				tabsHead.append('<a id="' + $this[0].id + '" href="#">' + $this.find('.gsection_title').html() + '</a>');
```

Comment:
So i dont actually know how to merge these changes into my pull request, but your suggestion works - no problems.